packages/core/content-releases/admin/src/components/CMReleasesInjectionZone.tsx

```diff
@@ -0,0 +1,265 @@
+        <Typography id="title" fontWeight="bold" textColor="neutral800">
+          {formatMessage({
+            id: 'content-releases.content-manager-edit-view.add-to-release',
+            defaultMessage: 'Add to Release',
+          })}
+        </Typography>
+      </ModalHeader>
+      <Formik
+        onSubmit={handleSubmit}
+        validationSchema={RELEASE_ACTION_FORM_SCHEMA}
+        initialValues={INITIAL_VALUES}
+      >
+        {({ values, setFieldValue }) => {
+          return (
+            <Form>
+              <ModalBody>
+                <Flex direction="column" alignItems="stretch" gap={2}>
+                  <SingleSelect
+                    required
+                    label={formatMessage({
+                      id: 'content-releases.content-manager-edit-view.add-to-release.select-label',
+                      defaultMessage: 'Select a releases',
+                    })}
+                    placeholder={formatMessage({
+                      id: 'content-releases.content-manager-edit-view.add-to-release.select-placeholder',
+                      defaultMessage: 'Select',
+                    })}
+                    onChange={(value) => setFieldValue('releaseId', value)}
+                    value={values.releaseId}
+                  >
+                    {releases?.map((release) => (
+                      <SingleSelectOption key={release.id} value={release.id}>
+                        {release.name}
+                      </SingleSelectOption>
+                    ))}
+                  </SingleSelect>
+                  <FieldLabel>
+                    {formatMessage({
+                      id: 'content-releases.content-manager-edit-view.add-to-release.action-type-label',
+                      defaultMessage: 'What do you want to do with this entry?',
+                    })}
+                  </FieldLabel>
+                  <Flex>
+                    <ReleaseActionOptions
+                      selected={values.type}
+                      handleChange={(e) => setFieldValue('type', e.target.value)}
+                    />
+                  </Flex>
+                </Flex>
+              </ModalBody>
+              <ModalFooter
+                startActions={
+                  <Button onClick={handleClose} variant="tertiary" name="cancel">
+                    {formatMessage({
+                      id: 'content-releases.content-manager-edit-view.add-to-release.cancel-button',
+                      defaultMessage: 'Cancel',
+                    })}
+                  </Button>
+                }
+                endActions={
+                  /**
+                   * TODO: Ideally we would use isValid from Formik to disable the button, however currently it always returns true
+                   * for yup.string().required(), even when the value is falsy (including empty string)
+                   */
+                  <Button type="submit" disabled={!values.releaseId} loading={isLoading}>
```

Comment:
ℹ️ Still looking into this. I don't what's up here but even if releaseId is undefined the validation still passes 🤔 . Currently even though the validation schema is passed to formik and runs it's basically useless....